Dockerfile.compute-node

```diff
@@ -0,0 +1,60 @@
+ARG COMPUTE_TOOLS_TAG=latest
```

Comment:
Will be used by follow-up merge: https://github.com/neondatabase/cloud/issues/1963 - currently inert.